orchagent/portsorch.cpp

```diff
@@ -5459,6 +5534,44 @@ void PortsOrch::generatePriorityGroupMap()
     m_isPriorityGroupMapGenerated = true;
 }
 
+void PortsOrch::removePriorityGroupMapPerPort(const Port& port)
+{
+    /* Remove the PG map in the Counter DB */
+    vector<string> pgVector;
+    vector<string> pgPortVector;
+    vector<string> pgIndexVector;
+
+    for (size_t pgIndex = 0; pgIndex < port.m_priority_group_ids.size(); ++pgIndex)
+    {
+        std::ostringstream name;
+        name << port.m_alias << ":" << pgIndex;
+
+        const auto id = sai_serialize_object_id(port.m_priority_group_ids[pgIndex]);
+
+        pgVector.emplace_back(name.str());
+        pgPortVector.emplace_back(id);
+        pgIndexVector.emplace_back(id);
+    }
+
+    m_counter_db->hdel(COUNTERS_PG_NAME_MAP, pgVector);
```

Comment:
m_pgPortTable->hdel() and m_pgIndexTable->hdel()?

Reply:
it's the same operation

on the deInitPort we also using removing in the same way:
m_counter_db->hdel(COUNTERS_PORT_NAME_MAP, alias);

Reply:
for consistency of ->set and ->hdel operations, we should use "m_pgPortTable" .  
`m_counter_db->hdel(COUNTERS_PORT_NAME_MAP, alias);` can be changed too if you want to address in this PR.

Reply:
@zhenggen-xu  , Code was fixed following your remark!